pkg/controller/kieapp/defaults/defaults.go

```diff
@@ -1015,6 +1024,31 @@ func getJmsConfig(environment api.EnvironmentType, jms *api.KieAppJmsObject) (*a
 	return jms, nil
 }
 
+func getKafkaConfig(kafka *api.KafkaExtObject) (*api.KafkaExtObject, error) {
+	if kafka == nil || !kafka.ExtEnabled {
+		return nil, nil
+	}
+
+	//if something is missing we set mandatory defaults
+	if kafka.MaxBlockMs == 0 {
```

Comment:
would making this a pointer and checking for nil be a better way to go about this?

Reply:
yep